File: comms/bulk-messaging/sender-pools/create-sender-pool/create-sender-pool.rb
require "twiliocomms"

client = TwilioComms::Client.new(
  account_id: "<username>",
  auth_token: "<password>"
)

client.sender_pools.create(
  name: "Sales Leads - APAC",
  tags: {
    region: "APAC"
  }
)

File: comms/bulk-messaging/sender-pools/create-sender-pool/create-sender-pool.php
<?php

namespace Example;

use Twilio\Comms\TwilioComms;
use Twilio\Comms\SenderPools\Requests\SenderPoolsCreateRequest;

$client = new TwilioComms(
    accountId: '<username>',
    authToken: '<password>',
);
$client->senderPools->create(
    new SenderPoolsCreateRequest([
        'name' => 'Sales Leads - APAC',
        'tags' => [
            'region' => 'APAC',
        ],
    ]),
);
